parts/k8s/kubernetesparams.t

```diff
@@ -288,7 +288,8 @@
     "cloudproviderConfig": {
       "type": "object",
       "defaultValue": {
-        "cloudProviderBackoff": true,
+        "cloudProviderBackoff": false,
+        "cloudProviderBackoffMode": "v1",
```

Comment:
Even though "v1" isn't actually a known value in cloudprovider code we need to provide a string value here in order to make the ARM params input validation happy.